LDSContent/ContentController.swift

```
@@ -217,8 +217,11 @@ extension ContentController {
             }
         }
         let mergedURL = URL(fileURLWithPath: mergedPath)
+        
         let directory = mergedURL.deletingLastPathComponent()
-        try FileManager.default.createDirectory(at: directory, withIntermediateDirectories: true, attributes: nil)
+        if !FileManager.default.fileExists(atPath: directory.path) {
```

That raises a good point. I don't really understand why its better to try to perform an operation and let it fail if its not possible, then react to the error. Like here, we don't want to create the directory if it already exists, so is it really better to try to create it, let it fail, then react to the error that's thrown?